addons/portal/__manifest__.py

```diff
@@ -39,18 +38,74 @@
         'web.assets_frontend': [
             'portal/static/src/scss/portal.scss',
             'portal/static/src/js/portal.js',
-            'portal/static/src/js/portal_chatter.js',
             'portal/static/src/xml/portal_chatter.xml',
             'portal/static/src/js/portal_composer.js',
             'portal/static/src/js/portal_security.js',
             'portal/static/src/js/portal_sidebar.js',
             'portal/static/src/xml/portal_security.xml',
             'portal/static/src/js/components/**/*',
             'portal/static/src/signature_form/**/*',
+            'portal/static/src/chatter/boot/boot_service.js',
         ],
         'web.assets_tests': [
             'portal/static/tests/**/*',
         ],
+        "portal.assets_chatter_helpers": [
+            "web/static/src/views/view_dialogs/form_view_dialog.js",
+            "web/static/src/core/debug/*",
+            "web/static/src/core/commands/command_hook.js",
+            "web/static/src/model/**/*",
+            "web/static/src/search/**/*",
+            "web/static/src/views/view.js",
+            "web/static/src/views/view_hook.js",
+            "web/static/src/webclient/actions/action_dialog.js",
+            "web/static/src/webclient/actions/reports/utils.js",
+            "web/static/src/webclient/actions/reports/report_action.js",
+            "web/static/src/webclient/actions/reports/report_hook.js",
+            "web/static/src/views/utils.js",
+            "web/static/src/views/fields/formatters.js",
+            "web/static/src/views/fields/file_handler.*",
+            "mail/static/src/model/**/*",
+            "mail/static/src/core/common/**/*",
+            "mail/static/src/core/web_portal/**/*",
+            "mail/static/src/utils/common/**/*",
+            "mail/static/src/chatter/web_portal/**/*",
+            "mail/static/src/discuss/typing/**/*",
```

Comment:
Should never import `**` (in the "bundle" slot) but `common` and whatever others are needed/existing.

But I'm not sure why we would need "discuss" (channel) stuff in portal bundle. Channel part should already be covered by livechat, when necessary.

Typing is only for channels, if you need it in chatter it means we messed up the split somewhere.